src/odemis/gui/cont/tabs.py

```diff
@@ -499,6 +501,29 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 self
             )
 
+    def _on_overview_visible(self, val):
+        """
+        Apply the visibility status of overview streams in the Localisation tab to
+        the overview streams in the Chamber tab.
+        """
+        # All the overview streams
+        acquired_streams = set()
+        acquired_streams.update(self.tab_data_model.overviewStreams.value)
+        # Visible overview streams
+        streams = self.tab_data_model.views.value[0].stream_tree.streams
+        list_streams = []
+        for st in streams:
+            list_streams.append(st.stream)
+        # Invisible overview streams
+        unused_streams = acquired_streams.difference(set(list_streams))
```

Comment:
I would probably rename these to for clarity

```python
all_ov_streams
visible_ov_streams
invisible_ov_streams

```

Reply:
I have changed the names of visible and invisible streams, it brings more clarity than before. For all_ov_streams for acquired_streams, I think as acquired_streams is used in function names and variable names in the other part of the file, consistency wise acquired_streams would bring more clarity. What do you think?